manifests/deploy/export.pp

```diff
@@ -18,8 +18,8 @@
   -> Openvpn::Client[$name]
   -> Openvpn::Deploy::Export[$name]
 
-  if $facts['openvpn'] {
-    if $facts['openvpn'][$server][$name] {
+  if has_key($facts, 'openvpn') {
```

Comment:
`has_key` is deprecated. https://github.com/puppetlabs/puppetlabs-stdlib#has_key
You'd normally replace it with `in`, but in this case (as @bastelfreak suggests), I think the `fact` function is probably best.